store/src/main/java/org/apache/rocketmq/store/queue/RocksDBConsumeQueue.java

```diff
@@ -480,4 +486,21 @@ public CqUnit nextAndRelease() {
             }
         }
     }
+
+    public void initializeWithOffset(long offset) {
```

Comment:
This is used for mount --> main in container mode? If so, add some java docs here.